src/components/Nav/Nav.js

```diff
@@ -1,7 +1,64 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
+import './Nav.scss';
+import './navData';
+import NAV_LIST from './navData';
+import NavList from './NavList';
 
 const Nav = () => {
-  return <div>nav</div>;
+  const [navActive, setNavActive] = useState(false);
+  const [scrollY, setScrollY] = useState(0);
+  const [scrollActive, setScrollActive] = useState(false);
+
+  const scrollFixed = () => {
+    if (scrollY > 100) {
+      setScrollY(window.pageYOffset);
+      setScrollActive(true);
+    } else {
+      setScrollY(window.pageYOffset);
+      setScrollActive(false);
+    }
+  };
+
+  useEffect(() => {
+    const scrollListener = () => {
+      window.addEventListener('scroll', scrollFixed);
+    };
+    scrollListener();
+    return () => {
+      window.removeEventListener('scroll', scrollFixed);
+    };
+  });
+  return (
+    <div className={'nav' + (scrollActive ? ' fixed' : '')}>
```

Comment:
```suggestion
    <div className={`nav${scrollActive ? ' fixed' : ''}`}>
```
위와 같은 방법으로도 작성할 수 있습니다! 
템플릿 리터럴도 잘 활용해 보세요:)